src/content/docs/en/guides/assets.mdx

```diff
@@ -50,7 +62,26 @@ import logo from '../assets/images/logo.png';
 <Image src={logo} alt="Astro" />
 ```
 
-### Content Collections integration
+## New Markdown support
+
+Images stored in `src/assets/` can be referenced in Markdown and MDX files. These will automatically be optimized and hashed by Astro's build process.
```

Comment:
Images do not need to be in the `assets` folder to be optimized, any images can be referred to